packages/Typography/src/Text.tsx

```diff
@@ -7,7 +7,7 @@ export type TextProps = HTMLAttributes<HTMLElement> &
         /** This will be the actual rendered element */
         as?: React.ReactNode;
         /** Change the size of the output. See the Theme provider for what these sizes will look like */
-        size?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
+        size?: 1 | 2 | 3 | 4 | 5 | 6 | 'body' | 'subhead' | 'body-small' | 'pre-heading' | 'table-heading';
```

Comment:
Wondering if this will bloat with more variants, e.g. "body-large", "post-heading", "table-heading-small", etc. Maybe worth considering a `variant` prop to hold these, then throw in a single `[styles[variant]]: true` classname. Just a thought. Regardless, this seems pretty useful.

Reply:
I see what you mean but I don't think separating "body" type styles from "headings" sizes works because you can't use both at the same time.